### PR TITLE
[feat] implemented custom OSD position

### PR DESCRIPTION
**Pull request by yuxiaolejs**
implemented custom OSD position by adding extra field in line 1267

### CODE REVIEW

**Comment by yuxiaolejs** [comment]
@agsh

**Comment by agsh** [comment]
@yuxiaolejs Hi! Nice PR! Please just add also jsdoc to describe all the possible parameters of `position` argument: https://github.com/yuxiaolejs/onvif/blob/2a803e33ab22f2a8cf04ab683db3863d27f1a3d6/lib/media.js#L1251

File: lib/media.js [hunk 1]
```diff
@@ -1263,7 +1263,8 @@ module.exports = function(Cam) {
             <sch:VideoSourceConfigurationToken>${(options.videoSourceConfiguationToken || this.activeSource.videoSourceConfigurationToken)}</sch:VideoSourceConfigurationToken>
             <sch:Type>Text</sch:Type>
             <sch:Position>
-               <sch:Type>${options.position || 'LowerLeft'}</sch:Type>
+               <sch:Type>${typeof (options.position) == "object" ? "Custom" : options.position}</sch:Type>
+               ${typeof (options.position) == "object" ? ("<sch:Pos x=\"" + options.position.x + "\" y=\"" + options.position.y + "\"/>") : ""}
```
Comment (agsh) [inline lib/media.js:1267]: And here you can use strict equal: `typeof (options.position) === 'object'`

**Comment by yuxiaolejs** [comment]
@agsh  Hi! I updated the docstring, and also changed it to strict equals. Have the code tested on my Hikvision DS-2CD3325-I IPC, everything worked as expected

**Comment by agsh** [comment]
@yuxiaolejs Cool! Should I put the updated version to NPM?

**Comment by agsh** [comment]
@yuxiaolejs Oh, and also fix lint errors, please: https://github.com/agsh/onvif/actions/runs/3729451314

**Comment by yuxiaolejs** [comment]
> @yuxiaolejs Cool! Should I put the updated version to NPM?

I am glad to hear that! It's my pleasure to contribute to this project, and seeing it updated on NPM is great!

**Comment by agsh** [comment]
@yuxiaolejs Great! Just make a new PR with all of the changes please (I've reverted the merge because the linting pipeline broke) :) And I'll do the rest.

**Pull request merged**